locale/fr/get-involved/index.md

```diff
@@ -3,44 +3,48 @@ title: Comment s'impliquer ?
 layout: contribute.hbs
 ---
 
-# Nous encourageons toute forme de contributions de la part de la communauté 
+# Nous encourageons toutes formes de contribution de la part de la communauté 
 
-La communauté Node.js est vaste, inclusive et heureuse de permettre à autant 
+La communauté Node.js est vaste, inclusive et entousiaste à l'idée de permettre à autant 
 d'utilisateurs de contribuer de toutes les manières possibles. Si vous souhaitez
 [signaler un problème](https://github.com/nodejs/node/issues), aider à documenter
 ou contribuer au code du projet, vous êtes au bon endroit. Explorez les ressources 
-de notre communauté pour trouver comment vous pouvez aider:
+de notre communauté pour trouver comment vous pouvez aider&nbps;:
 
 <hr>
 
-## Discussion de la communauté
+## Discussions de la communauté
 
-- La [liste des tickets GitHub](https://github.com/nodejs/node/issues) est le bon endroit pour discuter des fonctionnalités coeurs de Node.js.
+- La [liste des tickets GitHub](https://github.com/nodejs/node/issues) est le bon endroit pour discuter des fonctionnalités principales de Node.js.
 
-- Pour une discussion temps réel à propos du développement Node, allez sur `irc.freenode.net` dans le canal `#node.js` avec un [client IRC](https://fr.wikipedia.org/wiki/Liste_de_clients_IRC) ou connectez vous depuis votre navigateur web sur le canal [freenode's WebChat](http://webchat.freenode.net/?channels=node.js).
+- Pour une discussion temps réel à propos du développement Node, allez sur `irc.freenode.net` dans le canal `#node.js` avec un [client IRC](https://fr.wikipedia.org/wiki/Liste_de_clients_IRC) ou connectez vous depuis votre navigateur web en utilisant [le WebChat de freenode](http://webchat.freenode.net/?channels=node.js).
```

Comment:
Maybe we can mention that the IRC channel is English speaking to avoid confusion.